<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CloneDialog</class>
 <widget class="QDialog" name="CloneDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Clone a repository</string>
  </property>
  <property name="windowIcon">
   <iconset theme="kommit"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Url:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Directory:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="UrlRequester" name="lineEditPath" native="true"/>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="lineEditUrl"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBoxRecursive">
           <property name="text">
            <string>Recursive</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxDepth">
           <property name="text">
            <string>Depth</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QSpinBox" name="spinBoxDepth">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="maximum">
            <number>99999</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxBranch">
           <property name="text">
            <string>Branch</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="lineEditBranch">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxOriginName">
           <property name="text">
            <string>Origin name</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="lineEditOriginName">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>63</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="standardButtons">
          <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Cloning into:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCloneDestination">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelMessage">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>395</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UrlRequester</class>
   <extends>QWidget</extends>
   <header>widgets/urlrequester.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditUrl</tabstop>
  <tabstop>checkBoxDepth</tabstop>
  <tabstop>spinBoxDepth</tabstop>
  <tabstop>checkBoxRecursive</tabstop>
  <tabstop>checkBoxBranch</tabstop>
  <tabstop>lineEditBranch</tabstop>
  <tabstop>checkBoxOriginName</tabstop>
  <tabstop>lineEditOriginName</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxBranch</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditBranch</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>171</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxOriginName</sender>
   <signal>toggled(bool)</signal>
   <receiver>lineEditOriginName</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>44</x>
     <y>175</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxDepth</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinBoxDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>144</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CloneDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
